Fix vale warnings in End-user Guide (#2078)
codenvy/che-docs: modules/end-user-guide/partials/proc_troubleshooting-slow-workspaces.adoc

+
_Role: Administrator_
+

> Components of {prod-short} are OCI images. Setup {prod} in offline mode (air-gap scenario) to reduce any extra download at runtime as everything needs to be available from the beginning. See xref:installation-guide:installing-che-in-a-restricted-environment.adoc[].

Reviewer: ```suggestion
Components of {prod-short} are OCI images. Set up {prod} in offline mode (air-gap scenario) to reduce any extra download at runtime because everything needs to be available from the beginning. See xref:installation-guide:installing-che-in-a-restricted-environment.adoc[].
```

Reviewer: Noun: Setup
Verb: Set up
IBM SG:
```
as adverb
Do not use to mean "because" or "while".
Example: Write "Because you created the file, you can delete it", not "As you created the file, you can delete it".
```
